scripts/docs.sh

```diff
@@ -0,0 +1,19 @@
+cp ./README.md ./documentation/MAIN.md
+
+node ./scripts/make_version_file.js
+find ./documentation -type f > ./tree.txt
+node ./scripts/add_version_to_files.js
+
+passable_version=$(<version.txt)
+
+git config --global user.email $GIT_EMAIL
+git config --global user.name $GIT_NAME
+
+git clone -b $DOCS_BRANCH https://github.com/$GITHUB_REPO.git docs
+
+npx docpress b
```

Comment:
Consider adding `rm -rf docs/*` (does not delete hidden files like the `.git` directory) to get a fresh start and prevent leaving unlinked deprecated pages

Reply:
Good idea, thanks!